### ISSUE TITLE
Upgrade to newer Postgrex

### ISSUE THREAD

**Issue by scouten** [DavidAntaramian/tributary#10]
Tributary currently has dependency on Postgrex "~> 0.11.2" but Postgrex is now at 0.12.1.

**Comment by DavidAntaramian** [comment]
@scouten Yep! This can be done with #8 as well. Will handle it this week probably because I need it for work
